lib/Backend/LowerMDSharedSimd128.cpp

```diff
@@ -1327,7 +1336,11 @@ IR::Instr* LowererMD::Simd128LowerShift(IR::Instr *instr)
         IR::RegOpnd * shamtReg = IR::RegOpnd::New(TyInt8, m_func);
         shamtReg->SetReg(LowererMDArch::GetRegShiftCount());
         IR::RegOpnd * tmp = IR::RegOpnd::New(TyInt8, m_func);
-
+#ifdef _M_IX86
```

Comment:
Are we sure LinearScan tracks physical registers in a way that makes this safe? Do we understand why FilterRegIntSizeConstraints is not guaranteeing that we get a byteable register when we need one?